.changeset/search-otters-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend-node': minor
```

Comment:
Consider:

- Marking this (and the other similar changeset for ES) as `patch` instead of `minor`, given the API addition is on `plugin-search-common`, and there's otherwise no API changes to these two packages.
- Combining this and the similar changeset from Elastic into a single, generic changeset that just says something like...
  
  ```
  The search engine has been updated to take advantage of the `pageLimit` property on search queries. If none is provided, the search engine will continue to use its default value of 25 results per page.
  ```

Reply:
The changesets have been fixed. Is it as you suggested?